semcore/color-picker/src/index.d.ts

```diff
@@ -10,7 +10,7 @@ export type ItemProps = BoxProps & {
   /**
    * Color item in hexadecimal format.
    */
-  value?: string;
+  value?: string | null;
```

Comment:
Can this change affect users somehow? How does it related to storybook?

Reply:
this is real type, how we (or others) use it. 
for now, we check our code for examples with ts and that's why I added this.